eth/tracers/native/call.go

```diff
@@ -175,7 +179,12 @@ func (t *callTracer) CaptureState(pc uint64, op vm.OpCode, gas, cost uint64, sco
 			topics[i] = common.Hash(topic.Bytes32())
 		}
 
-		data := scope.Memory.GetCopy(int64(mStart.Uint64()), int64(mSize.Uint64()))
+		data, err := vm.GetMemoryCopyPadded(scope.Memory, int64(mStart.Uint64()), int64(mSize.Uint64()))
+		if err != nil {
+			// mSize was unrealistically large
+			return
```

Comment:
Not sure "silent return" is the best. IMO either errror-return or just continue with empty `data` ?

Reply:
The logs won't be emitted because execution of the op will fail.  So I silent-return to prevent emitting a log step in the tracer.